packages/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -76,6 +78,7 @@ export type GridCellProps = {
   onKeyDown?: React.KeyboardEventHandler<HTMLDivElement>;
   onDragEnter?: React.DragEventHandler<HTMLDivElement>;
   onDragOver?: React.DragEventHandler<HTMLDivElement>;
+  slotProps?: { tooltip?: Partial<TooltipProps> };
```

Comment:
Avoiding exposing material types on our public API is particularly important to achieve the design-system agnostic OKR :| If this is unavoidable, any way to create a minimal subset of the type to avoid exposing everything?